images/k8ssandra-system-logger/tests/01-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Could we also add a helm test? (and use the same chart in our README docs). The good news is we should largely be able to re-use this:

- https://github.com/chainguard-images/images/blob/1c3b1504b7a54fbc62f9ba397fd3666fc4a62f50/images/cass-operator/tests/helm.sh

But with different image override to the cass-operator. I think this might be what we'd need to override:
- https://github.com/k8ssandra/k8ssandra/blob/726158d32fcdd4a33f9fd6ee8dc4d84042bc906b/charts/cass-operator/values.yaml#L82
-

Reply:
> Can we add some more docs here on how to use the image?

Sorry - I should have more careful on that. (re: previous review)

> i.e perhaps with cass-operator helm operator (see comment below in test section) which seems to be the most common?

> Could we also add a helm operator test?

> But with different image override to the cass-operator.

This image is _actually_ nothing more than a `vector` package and a configuration file. We do not use `cass-operator` here, but only the config version, which is: `cass-operator-config`. So I think that our smoke test would be totally OK in this case. FWIW, we should add some test to our `vector` package, as a follow-up item. WDYT?

Reply:
It looks like the cass-operator itself though would be used with this image, i.e how customers would consume this. So was more to test the image with the operator the way customers would use it.

From looking through the links above it looks like the cass-operator deploys system-logger, so we'd want to have a test for that added. Do let me know if i'm missing anything though :)